backend/src/services/release.ts

```diff
@@ -55,7 +56,8 @@ async function validateRelease(user: UserInterface, model: ModelDoc, release: Re
     const fileNames: Array<string> = []
 
     for (const fileId of release.fileIds) {
-      let file
+      let file: Document<unknown, any, FileInterface> &
```

Comment:
This can be simplified to `let file: FileInterfaceDoc | undefined`

Reply:
Don't know why its saying I've done this, but I have changed anyway!